### PR TITLE
feat/supports

### PR DESCRIPTION
**Pull request by misspelz**
I integrated the endpoint for the amount raised, and implemented a circular progress bar that dynamically moves from zero to the current amount raised

### CODE REVIEW

**Comment by edustipend-root** [comment]
@misspelz, when you raise PRs please provide a description of the scope of the change in the PR so anyone on the team (including myself) who is reviewing this can know what we are looking at.

**Comment by misspelz** [comment]
Okay @meekg33k I'm sorry about that

File: src/sections/SupportALearner/Supports/Supports.jsx [hunk 1]
```diff
@@ -15,13 +14,49 @@ import {
...
+  const [displayedAmount, setDisplayedAmount] = useState(0);
+  const [progressPercentage, setProgressPercentage] = useState(0);
+  const [TotalAmountRaised, setTotalAmountRaised] = useState(0);
+
+  const innerStyle = {
+    background: `conic-gradient(#5801ff 0deg ${progressPercentage}%, #febd1c33 ${progressPercentage}deg 360deg)`
+  };
+
+  const fetchTransactions = useCallback(async () => {
+    const response = await getData(`donate/timeline`);
```
Comment (edustipend-root) [inline src/sections/SupportALearner/Supports/Supports.jsx:34]: @misspelz, we have an endpoint that provides you with the total amount already. You don't need to do any calculations on the front-end.

That endpoint is `/v1/donate/overview` - please work with @RB-Isiaq too as he is more familiar with the endpoint.

Comment (misspelz) [reply src/sections/SupportALearner/Supports/Supports.jsx:34]: Done

**Review by edustipend-root — APPROVED** [review]

**Comment by edustipend-root** [comment]
@misspelz, I fixed the failing tests so I'll be merging this in.

**Pull request merged**